pkg/sensors/exec/procevents/proc_reader.go

```diff
@@ -222,9 +234,6 @@ func writeExecveMap(procs []Procs) {
 			Pid:   kernelPid,
 			Ktime: 1,
 		},
-		Flags:  0,
-		Nspid:  0,
-		Buffer: 0,
 	})
```

Comment:
Why did you remove these initialisations? Maybe also initialise `Binary` and `Pad` to 0 as well.